source/plugins/ruby/in_kube_perfinventory.rb

```diff
@@ -71,6 +71,7 @@ def start
         @podCacheMutex = Mutex.new
         @thread = Thread.new(&method(:run_periodic))
         @watchPodsThread = Thread.new(&method(:watch_pods))
+        @@perfTelemetryTimeTracker = DateTime.now.to_time.to_i
```

Comment:
same here .. this can be instance variable (for example @watchPodsThread, @thread etc..)  and doesnt require to be class variable since this only used within this plugin.